modules/class-simplechart-wp-cli.php

```diff
@@ -0,0 +1,246 @@
+<?php
+
+/**
+ * Import Simplechart charts from a remote site via WP-CLI
+ */
+class Simplechart_WP_CLI extends WP_CLI_Command {
+
+	/**
+	 * @var string $_wpcom_api_url Templatized URL to retrieve a single post from the WordPress.com REST API
+	 */
+	private $_wpcom_api_url = 'https://public-api.wordpress.com/rest/v1.1/sites/{{site}}/posts/{{post}}';
+
+	/**
+	 * @var int|string|null $_author Author ID to assign imported posts to
+	 */
+	private $_author = null;
+
+	/**
+	 * @var string $_post_type The post type we want to work with
+	 */
+	private $_post_type = 'simplechart';
+
+	/**
+	 * @var array $_meta_keys List of post meta keys to set when creating a post
+	 */
+	private $_meta_keys = array(
+		'simplechart-data',
+		'simplechart-template',
+		'simplechart-chart-url',
+		'simplechart-chart-id',
+		'simplechart-errors',
+	);
+
+	/**
+	 * Imports Simplechart charts using the Jetpack/WPCOM REST API
+	 *
+	 * ## OPTIONS
+	 *
+	 * --site=<site_id>
+	 * : WPCOM site ID
+	 *
+	 * --posts=<post_ids>
+	 * : comma separated list of Simplechart post IDs on remote site
+	 *
+	 *
+	 * [--author=<author>]
+	 * : Assigns imported posts to a specific author ID or username
+	 *
+	 * ## EXAMPLES
+	 *
+	 * wp simplechart import_wpcom --site=123456 --posts=12,13,14 --author=6
+	 *
+	 * @synopsis --site=<site_id> --posts=<post_ids> [--author=<author_id>]
+	 */
+	function import_wpcom( $args, $assoc_args ) {
+		$post_ids = explode( ',', $assoc_args['posts'] );
+		$post_ids = array_map( 'absint', $post_ids );
+		$site_id = absint( $assoc_args['site'] );
+		$this->_set_author( $assoc_args );
+		WP_CLI::line( sprintf( __( "Starting import of %s posts from site %s\n", 'simplechart' ), count( $post_ids ), $site_id ) );
+		foreach ( $post_ids as $post_id ) {
+			WP_CLI::line( sprintf( __( 'Processing post %s', 'simplechart' ), $post_id ) );
+			$post_object = $this->_extract_wpcom_post_object( $site_id, $post_id );
+			if ( $post_object ) {
```

Comment:
`_extract_wpcom_post_object()` is a private method and we know it's gonna return `null` or a legit object. i am adding a method to check that `json_decode()` was :+1: tho